Library/Formula/mlkit.rb

```diff
@@ -0,0 +1,29 @@
+class Mlkit < Formula
+  desc "Compiler for the Standard ML programming language"
+  homepage "https://melsman.github.io/mlkit"
+  url "https://github.com/melsman/mlkit/archive/mlkit-4.3.8.tar.gz"
+  sha256 "337ea3d333c694871f05f9c3ecbf1c05d4bcd1a0993e87801507930a1637a5d4"
+
+  bottle do
+    sha256 "32bcf016846a33d8794230567d6b41ab7d703a13eff4568b318073ba2fe61426" => :el_capitan
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "mlton" => :build
+  depends_on :tex
```

Comment:
I think the solution is that I remove the tex dependency upstream and
provide a new revised formula. Please keep the issue open for a couple of
days until I have a fix...

Den fredag den 11. marts 2016 skrev Dominyk Tiller <notifications@github.com
>:

> In Library/Formula/mlkit.rb
> <https://github.com/Homebrew/homebrew/pull/49933#discussion_r55848887>:
>
> > @@ -0,0 +1,29 @@
> > +class Mlkit < Formula
> > +  desc "Compiler for the Standard ML programming language"
> > +  homepage "https://melsman.github.io/mlkit"
> > +  url "https://github.com/melsman/mlkit/archive/mlkit-4.3.8.tar.gz"
> > +  sha256 "337ea3d333c694871f05f9c3ecbf1c05d4bcd1a0993e87801507930a1637a5d4"
> > +
> > +  bottle do
> > +    sha256 "32bcf016846a33d8794230567d6b41ab7d703a13eff4568b318073ba2fe61426" => :el_capitan
> > +  end
> > +
> > +  depends_on "autoconf" => :build
> > +  depends_on "mlton" => :build
> > +  depends_on :tex
>
> I'm afraid not. The CI machines don't have tex installed so if it's
> anything other than a optional dependency we can't provide bottles for it,
> so we prefer to be very explicit about the :tex dependency by hosting
> them in their own tap.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/49933/files#r55848887>.
>